### PR TITLE
Doc/dashboard deploy

### LINKED ISSUE #13: Update Dashboard component doc

### PR DESCRIPTION
**Pull request by thiagoyeds**
Fix #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,102 @@
...
+          "docker_repository": "$docker_repository_processing_option2"
+        }
+      ]
+};
+```
+**Note: The SAPS scripts configured here must be the same as the Dispatcher component and Scheduler component**
 
-Desc: 
-Detailed informations for a specific region. This search can return more than one region detail. These information are get from SAPS service api.
+- [SAPS Dispatcher Service URL](/public/dashboardApp.js) allows the communication with the SAPS dispatcher. Assing the dispatcher address (```http://$dispatcher_access_ip:$dispatcher_access_port/```) into the ```urlSapsService``` property.
```
Comment (thiagomanel) [inline README.md]: not sure it is a matter of documentation only but also a matter of changing the code but is bad having different names for the same thing. For example, why we have both `dispatcher_access_port` and `submission_rest_server_port`? Shouldn't we use `dispatcher_access_port` in both places?

**Review by thiagomanel — CHANGES_REQUESTED** [review]
Thanks, @thiagoyeds. I've have made some direct commits to improve the documentation. I've also added a small request (note that, it is not a request to change the code, necessarily. I'm ok with an explanation).

File: README.md (hunk 1 above)
Comment (thiagoyeds) [reply README.md]: Yes, this is really bad, but it is something that already existed, what do you suggest? I can change it to ```http://$submission_rest_server_ip:$submission_rest_server_port/```

Comment (thiagomanel) [reply README.md]: bad. isn't clearer to user `http://$dispatcher_access_ip:$dispatcher_access_port/`?

Comment (thiagoyeds) [reply README.md]: Yes, I will create an issue in the saps-engine repository about changing this name to ```dispatcher_access_port```

**Pull request merged**